functions/jsonpath-value/1.0/index.js

```diff
@@ -0,0 +1,14 @@
+import jsonpath from './jsonpath.min';
+
+const jsonpathImplementation = async ({ data, path, outputType }) => {
```

Comment:
`jsonpathImplementation` is a kind of weird naming imo

Reply:
how should i call it? jsonpathFunction ? or jsonpathActionStep ?

Reply:
Now directly set it as the default export (so i dont need to set a name :D )